tests/func-tests/monitoring_test.go

```diff
@@ -68,8 +68,8 @@ var _ = Describe("[crit:high][vendor:cnv-qe@redhat.com][level:system]Monitoring"
 				if rule.Alert != "" {
 					Expect(rule.Annotations).To(HaveKeyWithValue("summary", Not(BeEmpty())),
 						"%s summary is missing or empty", rule.Alert)
-					Expect(rule.Annotations).To(HaveKeyWithValue("runbook_url", Not(BeEmpty())),
-						"%s runbook_url is missing or empty", rule.Alert)
+					Expect(rule.Annotations).To(HaveKeyWithValue("runbook_url", ContainSubstring(rule.Alert)),
+						"%s runbook_url is missing or doesn't contain alert name", rule.Alert)
```

Comment:
@assafad  What if a runbook_url will include the alert name but with a prefix or a suffix?
I think you need to remove the prefix and then check that the rest is equal to the alert name.

Reply:
I will submit a fix PR, but beware that we also check that the runbook URL is accessible, and it must be a suffix for that. And we are already add the `runbook_url` automatically:
https://github.com/kubevirt/hyperconverged-cluster-operator/blob/928efeddcff4b2c6481e7a3b0f843a7038c440e4/controllers/alerts/alerts.go#L149

Reply:
(but not across all repos)